rules-reviewed/azure/springboot/tests/spring-boot-to-azure-feign.windup.test.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruletest id="spring-boot-to-azure-feign-test" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <testDataPath>data</testDataPath>
```

Comment:
```suggestion
    <testDataPath>data/spring-boot-to-azure-feign</testDataPath>
```
Add the folder containing the test data to avoid inconsistencies in test if further test files will be added.

Reply:
Done